manual/source/classes/contributed_definitions/ellipsometry-structure.rst

```diff
@@ -14,42 +14,22 @@ Optical Spectroscopy
 Ellipsometry
 ##############
 
-Ellipsometry is an optical characterization method to describe optical properties of interfaces and thickness of films. The measurements are based on determining how the polarization state of light changes upon transmission and reflection. Interpretation is based on Fresnel equations and numerical models of the optical properties of the materials.
+Ellipsometry is an optical characterization method to describe optical properties of interfaces and thickness of films.
+The measurements are based on determining how the polarization state of light changes upon transmission and reflection.
+Interpretation is based on Fresnel equations and numerical models of the optical properties of the materials.
 
 In the application definition we provide a minimum set of description elements allowing for a reproducible recording of ellipsometry measurements. 
 
 
 Application Definitions
 -----------------------
 
-We created a generic application definition for every kind of Optical Spectroscopy experiments, and also a specialisation for Ellipsometry:
-
     :ref:`NXopt`:
        A generic application definition for optial spectorscopy measurements, including complex systems up to variable angle spectroscopic ellipsometry. 
 
     :ref:`NXellipsometry`:
-       A general application definition for ellipsometry measurements, including complex systems up to variable angle spectroscopic ellipsometry. 
-
-
-
-Base Classes Extended in Application Definitions
-------------------------------------------------
-
-We use existent base classes in application definitions and add descriptors:
-
-    :ref:`NXinstrument`
-       Added fields to add information that is important for an ellipsometry setup, such as the ellipsometer type, the light source, the type of the sample stage, or the angle(s) of incidence, as well as information on calibration, focussing probes, data correction etc. 
-              
-    :ref:`NXdetector`
-       Added fields to describe spectroscopic detection with polarization (e.g. rotating analyzer).
-       
-    :ref:`NXaperture`
-       Added fields to define parameters that describe windows (e.g. windows of a UHV cryostat), such as the thickness and the orientation angle of the window, as well as reference data to calculate window effects.
-       
-    :ref:`NXsample`
-       Added fields to specify the sample and material properties, as well as the sample environment (e.g. refractive index of surrounding medium) and experimental conditions (e.g. temperature, pressure, pH value etc.).
+       An application definition for ellipsometry measurements, including complex systems up to variable angle spectroscopic ellipsometry. 
 
-.. _DispersiveMaterial:
```

Comment:
we need the index as it is defined above

Reply:
Not sure what you are referring to

Reply:
this line is an anchor which we shall not remove to enable url to point directly to this subsection

Reply:
aah I see. This was an accident actually...